### DISCUSSION TITLE
Your printer and file format seems only to support the following versions: (517).

### DISCUSSION THREAD

**Discussion by w696969** [sn4k3/UVtools#931] [Q&A] [answered]
Hello! What does this warning mean? What versions are we talking about? I bought Аnicubic Photon Mono  M5 printer and I started having problems. UV Tools distorts the geometry along the Y axis by 26%
And I can't solve this problem.
Your printer and file format seems only to support the following versions: (517). However, the file comes with the version 516 set, which is outside the supported range.
Keeping that version may or not print on your machine, but can mislead UV tools to set and use features that your printer firmware doesn't support or vice versa leading to unexpected behaviors.

Do you want to change the version 516 to the latest supported version 517?
Yes: Change to version 517. (Highly recommended!)
No: Keep the version 516 while editing, but a latter full encode of the file will force the version 517.

**Comment by sn4k3** [comment, marked as answer]
File formats schema may have some iterations and then different versions which firmware may only know one or some. In teory upper versions should be backward compatible but most printers still not print them. A lower version than printer firmware has implemented often does work. As so, this is security feature to be sure the file uses proper version for the printer. As text says it can or can not work, depends on your printer, but the suggestions is to always use the version aimed for that printer. As also recommended to opt for accept the change "Yes".

So I guess you are not using photon workshop? But Chitubox?



> UV Tools distorts the geometry along the Y axis by 26%

That happen on printed object or in software layer preview inside UVtools?

**Comment by w696969** [comment]
Yes, I use Chitubox. Distortions occur during the preview, inside UVtools.
But when I upload the file back to Chitubox to create the right number of copies, distortions occur during printing. The problem arose when I started using the Anicubic Photon Mono M5 printer,
the M3 and x6s models had no such problems. I do not know how to work now, how to clone and print models.

**Reply by sn4k3** [reply]
- Chitubox does not follow same norms as anycubic slicer, if you use anycubic slicer it will set the versions correctly.
- Distortion in preview is normal and expected, is due your pixel size and will print correctly. https://github.com/sn4k3/UVtools/discussions/913
- Never ever reimport CTB into slicer to reslice! (This will degrade model quality and problems) Do the copies on UVtools: Tool - Pattern. ([More](https://www.facebook.com/groups/uvtools/posts/1166433801132788/?comment_id=1166535381122630&reply_comment_id=1166960114413490))

**Reply by w696969** [reply]
Thanks! I couldn't figure out how to copy a file, how to set the size of the table. If there is a lesson, please give a link to it.

**Reply by sn4k3** [reply]
There are no video, you simply choose the grid (Number of rows and columns), eg: 4 x 4 = 16 copies
